fastlane/lanes/build.rb

```diff
@@ -271,6 +285,14 @@
       project_slug: SENTRY_PROJECT_SLUG_WORDPRESS,
       dsym_path: lane_context[SharedValues::DSYM_OUTPUT_PATH]
     )
+
+    upload_gutenberg_sourcemaps(
+      sentry_project_slug: SENTRY_PROJECT_SLUG_WORDPRESS,
+      release_version: release_version_current_internal,
+      build_version: build_code_current_internal,
+      app_identifier: 'org.wordpress.internal'
+    )
```

Comment:
This source map will be used for the internal builds of the WordPress app. As far as I checked, this version is seems no longer used.